app/assets/javascripts/modules/MultiTableFilter.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * # Multiple table filter
+ *
+ * Filters across one or more tables, and can also hide the elements associated with a table.
+ *
+ * Requires an element to have a data-dough-component="MultiTableFilter" attribute and an ID.
+ *
+ * Makes use of jquery.fastLiveFilter.
+ * See test fixture for sample markup - /spec/js/fixtures/MultiTableFilter.html
+ */
+
+define(['jquery', 'jqueryFastLiveFilter', 'DoughBaseComponent'],
+       function($, jqueryFastLiveFilter, DoughBaseComponent) {
+  'use strict';
+
+  var MultiTableFilterProto,
+      defaultConfig = {
+        filterTargetSelector: '.js-filterable',
```

Comment:
Yes yes, that's usually my preference too — the old JS library didn't support that, hence the classes, but no reason we shouldn't change it now.